docs/data/charts/composition/composition.md

```diff
@@ -105,6 +105,41 @@ The order of elements in composition is the only way to define how they overlap.
 
 To display data, you have components named `<XxxPlot />` such as `<LinePlot />`, `<AreaPlot />`, `<MarkPlot />`, `<BarPlot />`, etc.
 
+### Clipping
+
+To ensure chart elements stay confined to the designated drawing area, leverage the `ChartsClipPath` component.
```

Comment:
Maybe that's just me, but `leverage` is not a super common word and we don't use it a lot.
Since we target an international audience, sticking to simple (boring :laughing: ) words when they don't loose any subtlety to the sentence might be better.

I would stick to the good old "you can use" here.